src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -1454,15 +1454,35 @@ citationsToBiblatex
                   NormalCitation -> "autocite"
 
 citationsToBiblatex (c:cs)
-  | all (\cit -> null (citationPrefix cit) && null (citationSuffix cit)) (c:cs)
+  | all (\cit -> null (citationPrefix cit) && null (citationSuffix cit)) 
+    (c:cs)
     = do
       let cmd = case citationMode c of
                     SuppressAuthor -> "\\autocite*"
                     AuthorInText   -> "\\textcite"
                     NormalCitation -> "\\autocite"
       return $ text cmd <>
                braces (literal (T.intercalate "," (map citationId (c:cs))))
-  | otherwise = do
+  | all (\cit -> null (citationSuffix cit)) (c:cs) 
+    = do  
+      let cmd = case citationMode c of 
+                    SuppressAuthor -> "\\autocites*"
+                    AuthorInText   -> "\\textcites"
+                    NormalCitation -> "\\autocites"
+
+      let pfxs = tail $ scanl (\prev cit -> let pfx = citationPrefix cit in
```

Comment:
`tail` is a partial function and should be avoided generally.  It can cause an exception if the argument is an empty list.